consul-1.16.yaml

```diff
@@ -1,11 +1,14 @@
 package:
-  name: consul
+  name: consul-1.16
   version: 1.16.0
   # When bumping the version check if the CVE/GHSA mitigations below can be removed.
-  epoch: 1
+  epoch: 2
```

Comment:
Wasn't sure if this was needed because we have `consul-1.16.0-r1` already.  Maybe this can be `0` and the `provides` should have `.99`?

Reply:
Yes, the `.999` is needed when transitioning to a version stream, but should otherwise not be done.

Reply:
Done!